homeassistant/components/lutron_caseta/config_flow.py

```diff
@@ -31,22 +49,129 @@ class LutronCasetaFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     def __init__(self):
         """Initialize a Lutron Caseta flow."""
         self.data = {}
+        self.lutron_id = None
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        if user_input is not None:
+            self.data[CONF_HOST] = user_input[CONF_HOST]
+            return await self.async_step_link()
+
+        return self.async_show_form(step_id="user", data_schema=DATA_SCHEMA_USER)
```

Comment:
await?

Reply:
Its decorated with `@callback` to indicate it is safe to run in async context (and if we ever pass it to hass.async_add_job it will run in async).